package.json

```diff
@@ -104,6 +104,7 @@
     "babel-plugin-transform-object-assign": "^6.22.0",
     "core-js": "^2.4.1",
     "gulp-sourcemaps": "^2.6.0",
-    "just-clone": "^1.0.2"
+    "just-clone": "^1.0.2",
+    "natives": "^1.1.4"
```

Comment:
Why was this `natives` package added in?  It should be removed if there's no need for it.